include/container.h

```diff
@@ -81,6 +82,11 @@ struct sway_container {
 	char *class;
 	char *app_id;
 
+	// Used by output containers to keep track of swaybar/swaybg child
+	// processes.
+	list_t *bar_pids;
+	pid_t bg_pid;
+
```

Comment:
I don't know if this is the best approach, but what about adding a `void*` field to `swayc_t` which can be used for whatever, in this case it would point to:

```
struct swayc_output {
    list_t *bar_pids;
    pid_t bg_pid;
    // other fields unique to a swayc_t of type C_OUTPUT
};
```